guides/source/command_line.md

```diff
@@ -124,6 +93,29 @@ development:
 
 It generated a database configuration corresponding to our choice of PostgreSQL.
 
+### Skipping Defaults
+
+If you wish to skip some files from being generated or skip some libraries
+entirely, you can pass one of the `--skip` arguments to the `rails new` command:
+
+```bash
+$ rails new sas --skip-active-storage
+Based on the specified options, the following options will also be activated:
+
+  --skip-action-mailbox [due to --skip-active-storage]
+  --skip-action-text [due to --skip-active-storage]
+
+      create
+      create  README.md
+      ...
+```
+
+In the above example, Action Mailbox and Action Text are skipped in addition to
+Active Storage because they depend on Active Storage functionality.
+
+For a full list of options (including what can be skipped), run
+`rails new --help`.
```

Comment:
Can you make this a full bash example command? No need for the output though. 
That makes it stand out more and easier to copy.